### DISCUSSION TITLE
Why is celery library named celery

### DISCUSSION THREAD

**Discussion by guanicoe** [celery/celery#7023] [Q&A]
This is a stupid question, but it's not in the documentation (as far as far as i could see). Why was the name celery chosen for this library. Although definitely not a priority, this should be in the FAQ at least.

Thanks

**Comment by jshields** [comment]
FWIW, I have always heard it's called Celery because it is fed to RabbitMQ

**Comment by frost-nzcr4** [comment]
It accidentally correlates to increased [vegan trends](https://trends.google.com/trends/explore?date=all&q=vegan) in Google either. Coincidence?

**Comment by guanicoe** [comment]
where do you see the correlation? the link only shows vegan, celery python is near 0

**Comment by dejlek** [comment]
As far as I understand, the original author (Ask Solem) preferred to use vegetable names for projects at the time Celery project started. GitHub: @ask

**Comment by clokep** [comment]
> preferred to use vegetable names for projects

See also carrot, the precursor to kombu: https://github.com/ask/carrot

**Comment by auvipy** [comment]
rabbit consume celery and carrot ;)
